src/proto/robot_status_msg.proto

```diff
@@ -40,12 +40,12 @@ message JetsonStatus
 message ChipperKickerStatus
 {
     // Indicates the time elapsed since chipper was last fired in ms
-    // UINT64_MAX indicates chipper was never fired
-    uint32 ms_since_chipper_fired = 1;
+    // UINT32_MAX indicates chipper was never fired
```

Comment:
We can remove the comment now (or update it if there's some custom intialization that we do (same with other similar comments in this file)